modules/cluster-cloud-controller-config-osp.adoc

```diff
@@ -0,0 +1,296 @@
+
+// This option can be useful if you have multiple or dual-stack interfaces attached to a node that need a user-controlled, deterministic way of sorting addresses.
+// |===
+
+[id="ccm-config-lb-options"]
+== Load balancer options
+
+CCM supports several load balancer options for deployments that use Octavia. 
+
+[NOTE]
+====
+Neutron-LBaaS support is deprecated. 
+====
+
+// Commented options are untested as of yet.
+
+|===
+| Option | Description
+
+| `enabled`
+| Whether or not to enable the `LoadBalancer` type of services integration. The default value is `true`.
+
+// Removed: always enforced.
+//| `use-octavia`
+//| Whether or not to use Octavia for the `LoadBalancer` type of service implementation rather than Neutron-LBaaS. The default value is `true`.
+
+| `floating-network-id`
+| Optional. The external network used to create floating IP addresses for load balancer virtual IP addresses (VIPs). If there are multiple external networks in the cloud, this option must be set or the user must specify `loadbalancer.openstack.org/floating-network-id` in the service annotation.
+
+| `floating-subnet-id`
+| Optional. The external network subnet used to create floating IP addresses for the load balancer VIP. Can be overridden by the service annotation `loadbalancer.openstack.org/floating-subnet-id`.
+
+| `floating-subnet`
+| Optional. A name pattern (glob or regular expression if starting with `~`) for the external network subnet used to create floating IP addresses for the load balancer VIP. Can be overridden by the service annotation `loadbalancer.openstack.org/floating-subnet`. If multiple subnets match the pattern, the first one with available IP addresses is used.
+
+| `floating-subnet-tags`
+| Optional. Tags for the external network subnet used to create floating IP addresses for the load balancer VIP. Can be overridden by the service annotation `loadbalancer.openstack.org/floating-subnet-tags`. If multiple subnets match these tags, the first one with available IP addresses is used.
+
+If the {rh-openstack} network is configured with sharing disabled, for example, with the `--no-share` flag used during creation, this option is unsupported. Set the network to share to use this option.
+
+| `lb-method`
+| The load balancing algorithm used to create the load balancer pool. The value can be `ROUND_ROBIN`, `LEAST_CONNECTIONS`, or `SOURCE_IP`. The default value is `ROUND_ROBIN`.
+
+For the OVN provider, only the `SOURCE_IP` algorithm is supported.
+
+For the Amphora provider, to use the `LEAST_CONNECTIONS` or `SOURCE_IP` methods, you must configure the `"create-monitor=true` option in the `cloud-provider-config` namespace and `ETP:Local` on the load balancing service.
```

Comment:
I think we can rephrase this to something like (to make it less mandatory):
```suggestion
For the Amphora provider, when using the `LEAST_CONNECTIONS` or `SOURCE_IP` methods, it's recommended to configure the `"create-monitor=true` option in the `cloud-provider-config` configMap on the `openshift-config` namespace and `ETP:Local` on the load balancer type service in order to get the balancing algorithm enforcement in the client to service endpoint connections.
```

Reply:
@eurijon Is this the same idea? 

```suggestion
For the Amphora provider, if using the `LEAST_CONNECTIONS` or `SOURCE_IP` methods, configure the `"create-monitor=true` option in the `cloud-provider-config` configMap on the `openshift-config` namespace and `ETP:Local` on the load-balancer type service to allow balancing algorithm enforcement in the client to service endpoint connections.
```

Reply:
@maxwelldb sure it is